src/ui/zcl_abapgit_gui_page_repo_view.clas.abap

```diff
@@ -149,6 +149,8 @@ CLASS zcl_abapgit_gui_page_repo_view DEFINITION
         !io_tb_branch     TYPE REF TO zcl_abapgit_html_toolbar
         !io_tb_tag        TYPE REF TO zcl_abapgit_html_toolbar
         !io_tb_advanced   TYPE REF TO zcl_abapgit_html_toolbar
+        !io_tb_stage      TYPE REF TO zcl_abapgit_html_toolbar
+        !io_tb_export     TYPE REF TO zcl_abapgit_html_toolbar
```

Comment:
there are two new toolbars? I'd expect only one for the stage by filter functionality?